### PR TITLE
refactor: filter networks with new VHS API

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

This PR filters nodes based on network using the new VHS API filters.

### Context of Change

https://github.com/ripple/validator-history-service/pull/32

NOTE: This PR should not be merged/deployed until the changes to the VHS have been deployed

### Type of Change

- [x] Refactor (non-breaking change that only restructures code)

### TypeScript/Hooks Update

N/A

- [ ] Updated files to React Hooks
- [ ] Updated files to TypeScript

## Before / After

No real change, though with the new VHS updates, more validators show up on the proper networks.

## Test Plan

CI passes. Works locally, with the VHS staging API endpoints.

### CODE REVIEW

File: server/routes/v1/nodes.js [hunk 1]
```diff
@@ -1,7 +1,12 @@
 const { default: axios } = require('axios');
 const log = require('../../lib/logger')({ name: 'nodes' });
 
-const DATA_URL = process.env.REACT_APP_DATA_URL;
+const ENV_NETWORK_MAP = {
+  mainnet: 'main',
+  testnet: 'test',
+  devnet: 'dev',
+  'nft-devnet': 'nft-dev',
```
Comment (JST5000) [inline server/routes/v1/nodes.js:8]: Why is `nft-devnet`'s key a string when the other ones have no quotes?

File: server/routes/v1/validators.js [hunk 2]
```diff
@@ -1,49 +1,29 @@
...
-      ) {
-        cache.validators = cache.validators.filter(v => v.unl === process.env.REACT_APP_VALIDATOR);
-      } else {
-        cache.validators = cache.validators.filter(
-          v =>
-            v.agreement_30day &&
-            v.agreement_30day.score &&
-            (v.unl === process.env.REACT_APP_VALIDATOR || v.unl === false)
-        );
-      }
```
Comment (JST5000) [inline server/routes/v1/validators.js:43]: Is the filtering now happening on the api side so we don't need to manually check here? 
(In other words, why was this deleted?)

File: server/routes/v1/nodes.js (hunk 1 above)
Comment (ckniffen) [reply server/routes/v1/nodes.js:8]: Having a dash requires quotes in JavaScript since it is an invalid variable character.

File: server/routes/v1/validators.js (hunk 2 above)
Comment (mvadari) [reply server/routes/v1/validators.js:43]: Yes - see https://github.com/ripple/validator-history-service/pull/32

**Review by khancode — APPROVED** [review]
